{"name": "test_login_with_invalid_credentials", "status": "passed", "steps": [{"name": "Logging in with invalid credentials", "status": "passed", "steps": [{"name": "Logging response", "status": "passed", "attachments": [{"name": "Response Text", "source": "18ed1103-a94d-4e0f-a5e7-ad8e2e243288-attachment.json", "type": "application/json"}, {"name": "Response Headers", "source": "bdd24ac8-42ae-4e09-8c50-01684d86450a-attachment.json", "type": "application/json"}], "start": 1719520598527, "stop": 1719520598528}], "start": 1719520597466, "stop": 1719520598528}], "start": 1719520597466, "stop": 1719520598528, "uuid": "e680e8d2-074b-4066-9eb3-46edceedc9f6", "historyId": "f20cbba362e81ee795bd81cc27c692ff", "testCaseId": "f20cbba362e81ee795bd81cc27c692ff", "fullName": "tests.test_user_operations.TestUser#test_login_with_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('user_page', 'user_data', 'created_user')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "14792-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}